src/main/java/core/basesyntax/MainApp.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.FigureSupplier;
+
+public class MainApp {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        final int figures = 6;
```

Comment:
it's should be constant

Reply:
final int figures = 6; This one?? It's a constant.

Reply:
+